lm_eval/tasks/boxes/boxes-base.yaml

```diff
@@ -0,0 +1,33 @@
+task: boxes
+dataset_path: json
+dataset_name: null
+dataset_kwargs:
+  data_files: {'test': 'test-subsample-states-t5.jsonl'}
```

Comment:
This points to a local file?

Reply:
Yes, I downloaded dataset from git, it is not on hugging face. How should I go about it excluding uploading data on hugging face?
I can try to confirm only flan-t5-xl results (since GPT3, 3.5, 4 are not an option). Please let me know if I understood it correctly :) Thank you in anticipation

Reply:
Hi Hailey, apologies for delay
I asked Dr. Sebastian Schuster and create this gated repo [Boxes task](https://huggingface.co/datasets/irafayabdul/boxes)

Further, I am attaching results for flan-t5 base and xl which are almost same as the results presented in [paper](https://aclanthology.org/2023.acl-long.213.pdf). 
here I tried to produce results based on number of operations affecting box state just as reported in paper but the task does not necessary need this in general hence implementation is kept same.

![flan-pop](https://github.com/EleutherAI/lm-evaluation-harness/assets/29028477/1cb384cb-a46a-4335-9e1d-8df7168f1763)
![flan-test](https://github.com/EleutherAI/lm-evaluation-harness/assets/29028477/dccb0ef9-9b12-484a-88d1-963fc2d3271f)